src/templates/dashboard.html

```diff
@@ -98,91 +110,67 @@ <h2></h2>
+				<th>Purpose</th>
 				<th>Weighting</th>
 				<th>Methodology</th>
 			</tr>
 			<tr>
 				<td>1. Humanitarian</td>
+				<td>To assess the use of the specific humanitarian elements added to the IATI Standard at v2.02 </td>
 				<td>25%</td>
 				<td><a href="http://dashboard.iatistandard.org/humanitarian.html#h_narrative">Methodology</a></td>
 			</tr>
 			<tr>
 				<td>2. Timeliness</td>
+				<td>To assess both the frequency (how often the data is updated) and the time lag (how up to date the data is) of published information </td>
 				<td>25%</td>
 				<td><a href="http://dashboard.iatistandard.org/timeliness.html#h_narrative">Methodology</a></td>
 			</tr>
 			<tr>
 				<td>3. Comprehensiveness</td>
+				<td>To assess how much of the IATI Standard is being used</td>
 				<td>25%</td>
 				<td><a href="http://dashboard.iatistandard.org/comprehensiveness.html#h_narrative">Methodology</a></td>
 			</tr>
 			<tr>
 				<td>4. Coverage</td>
+				<td>To assess the percentage of an organisation's total operational spend on all humanitarian programming that is published to IATI</td>
 				<td>15%</td>
 				<td><a href="http://dashboard.iatistandard.org/coverage.html#h_narrative">Methodology</a> Additional information will be required from publishers (see above)</td>
 			</tr>
 			<tr>
 				<td>5. Forward looking</td>
+				<td>To assess how much information on activity budgets is available for the next three years </td>
 				<td>10%</td>
 				<td><a href="http://dashboard.iatistandard.org/forwardlooking.html#h_narrative">Methodology</a></td>
 			</tr>
 		</table>
+		
+		<h3>What is classified as humanitarian data?</h3>
+		
+		<p>An organisation’s data is classed as <i>humanitarian</i> if either of the following applies:</p>
 
-		<h3>Understanding the dashboard</h3>
-
-		<p>
-			<strong>Baseline:</strong> the total value for each Grand Bargain signatory organisation as of 1 June 2017.<br/>
-			<strong>Total:</strong> the total value for each Grand Bargain signatory organisation at that point in time.<br/>
-			<strong>Progress:</strong> the difference between the organisation's <strong>total</strong> and <strong>baseline</strong> value. Any value greater than zero is coloured green.
-		</p>
-
-		<p>An organisation's data will be classed as <strong>humanitarian</strong> if either of the following apply:</p>
 		<ul>
-			<li>The activity includes a <a href="http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/sector/">DAC sector code</a> in the range 72010 to 74010.</li>
-			<li>The <a href="http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/">activity humanitarian attribute</a> is set. (Note that the humanitarian attribute is only available to publishers using V2.02 or later of the IATI Standard.)</li>
+			<li>The activity includes a <a href="http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/sector/">DAC sector</a> code in the range 72010 to 74010. </li>
```

Comment:
Note the Grand Bargain [logic for determining what is a humanitarian activity](https://github.com/devinit/gbm-IATI-Stats/blob/include-humanitarian-transactions/statsrunner/loop.py#L61-L109) also takes account of uses of the `SectorCategory` as well as `Sector` codes.

Reply:
Also suggest making the link to the codelist version independent:
`http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/sector/` -> `http://iatistandard.org/activity-standard/iati-activities/iati-activity/sector/`

In this case it will redirect the latest version of the Standard.